flink/src/test/java/org/apache/iceberg/flink/TestFlinkTableSource.java

```diff
@@ -40,14 +40,12 @@
   private static final String CATALOG_NAME = "test_catalog";
   private static final String DATABASE_NAME = "test_db";
   private static final String TABLE_NAME = "test_table";
-  private final String expectedFilterPushDownExplain = "FilterPushDown";
```

Comment:
the new table source interface remove the method `explainSource` so that we can not override the method  to rewrite the explain,  so I remove the explain assertion, only use the scan event listener to do the assertion,I think it is enough to assert filter push down and verify correctness, and it will also make the code simpler.

Reply:
Yeah,  that sounds reasonable.  the `lastScanEvent.filter().toString()` is enough to validate the pushed down filters.